app/views/admin/words/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% provide(:title, "Category words") %>
+
+<div class="container mt-4">
+    <div class="row justify-content-center">
+        <div class="card shadow col-md-8">
+            <div class="card-body">
+                <h1 class="mt-4"><%= @category.title %></h1>
+                <hr>
+                <table class="table table-hover">
+                    <thead>
+                        <tr>
+                            <th>Word</th>
+                            <th>Answer</th>
+                            <th>Action</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <% @words.each do |word| %>
+                        <tr>
+                            <td><%= word.name %></td>
+                            <td>answer</td>
+                            <td>
+                                <%= link_to "Edit", edit_admin_category_word_path(@category.id, word.id) %> | <%= link_to "Delete", admin_category_word_path(@category.id, word.id), method: :delete,
```

Comment:
@shaniajubelag 
- Please provide proper line spacing between elements, so it's more readable.
- In your edit button route, you can omit the `id`, so you can just pass `@category` and `word`
- In your delete button route, you can use a array format like this `[:admin, @category, word]`

You can read more here : https://guides.rubyonrails.org/routing.html#creating-paths-and-urls-from-objects
(Example)
![image](https://user-images.githubusercontent.com/20507565/61676721-95372c80-ad2f-11e9-86e1-9f3c89720b40.png)